docs/content/modeling/advanced/overview.mdx

```diff
@@ -9,7 +9,7 @@ import { CardGrid, DocumentationNotice, IntroCard, ProductName, ProductNameForma
 
 <DocumentationNotice />
 
-This section will present advanced use cases and patterns for authorization modelling with <ProductName format={ProductNameFormat.LongForm}/>
+This section will present advanced use cases and patterns for authorization modelling with <ProductName format={ProductNameFormat.LongForm}/>.
```

Comment:
Nitpick: we should use the same spelling 'modeling' or 'modelling' everywhere, and it seems like now most of our headings are 'modeling'.

Reply:
Done